src/elements/frame-router.ts

```diff
@@ -103,6 +129,33 @@ class FrameRouterElement extends HTMLElement {
       this.changeRoute(newValue);
     }
   }
+
+  private _handleClientMessage(message: ClientToHost): void {
+    switch (message.msgType) {
+      case 'publish':
+        this._subscriptionManager.dispatchMessage(message.msg);
```

Comment:
I had to change this up from the parent since we weren't actually using subscription manager to determine if we needed to dispatch this message.